lib/elastic_record/connection.rb

```diff
@@ -37,10 +38,10 @@ def json_delete(path, json = nil)
     end
 
     def json_request(method, path, json)
-      body = json.is_a?(Hash) ? ActiveSupport::JSON.encode(json) : json
+      body = json.is_a?(Hash) ? Oj.dump(json, :mode => :compat) : json
       response = http_request_with_retry(method, path, body)
 
-      json = ActiveSupport::JSON.decode response.body
+      json = Oj.compat_load(response.body)
```

Comment:
OK.  I ask because I know that Oj can demarshall arbitrary objects if it's not configured properly, as the subscription bot was doing.  I can't figure out if compat mode is susceptible to this from the documentation.